LL-7404 - SWAP - Increase fixed rate timing to 60s

jgiamporcaro-ledger opened the pull request:
## 🦒 Context (issues, jira)

[LL-7404]

## 💻  Description / Demo (image or video)

<img width="1440" alt="Screenshot 2021-09-24 at 15 42 40" src="https://user-images.githubusercontent.com/33158502/134684515-c14bc24d-3a1e-4f50-886f-f1f1813d1896.png">


<!-- please attached an image or even better a video that demo what this PR do -->

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: <!-- NB: Delivery incrementally with feature flagging is better than a very long PR. so prefer Option 1 if Option 2 takes more than a sprint -->
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

<!--
If expectations aren't met, please document it carefully (on the reason you can't check it) and what do you need from maintainers.
-->


[LL-7404]: https://ledgerhq.atlassian.net/browse/LL-7404

jelbaz-ledger reviewed (approved):
LGTM 👍

juan-cortes reviewed (commented):
The reason I said 45 was that if edge it to 60s, anything that's close to it will 100% result in expired rates on the summary screen. But why not, let's see how this goes

jgiamporcaro-ledger commented:
@juan-cortes I added a message to the ticket days ago to explain that we added a safety margin and to ask the product team if they are 100% sure about this change but they don't answer me 🤷
